stdlib/Cargo.toml

```diff
@@ -32,6 +32,7 @@ with-debug-info = []
 
 [dependencies]
 assembly = { package = "miden-assembly", path = "../assembly", version = "0.10", default-features = false }
+once_cell = "1.19.0"
```

Comment:
This should be `default_features = false` and include the `race` feature, right?